### ISSUE TITLE
[Accessibility] Notes should have a lighten gray in the dark mode.

### ISSUE THREAD

**Issue by MenaiAla** [react-navigation/react-navigation.github.io#1010]
When I was reading the documentation in the dark mode I've noticed that it was hard for my eyes to read the notes or the explanations because of the chosen shade for the grey.

#### Screenshot

<img width="819" alt="Screen Shot 2021-06-16 at 10 43 24 PM" src="https://user-images.githubusercontent.com/20143684/122298403-a0cc0780-cef4-11eb-97f4-6460af6af44b.png">

#### Link

[Hello React Navigation](https://reactnavigation.org/docs/hello-react-navigation)

#### Suggestion

I'd recommend choosing a light gray to make the text of notes more readable for people with low vision like me.
